components/new_post/post_component.tsx

```diff
@@ -382,7 +383,7 @@ const PostComponent = (props: Props): JSX.Element => {
             <PostAriaLabelDiv
                 ref={postRef}
                 role='listitem'
-                id={'rhsPost_' + post.id}
+                id={props.location === 'RHS_ROOT' ? `rhsPost_${post.id}` : 'postView'}
```

Comment:
Seeing actual messages in community, perhaps ```id={props.location === 'RHS_ROOT' ? `rhsPost_${post.id}` : `post_${post.id}`}``` along with adding a `data-testid="postView"` (currently this is in place) would fix several of the e2e test errors I think